pkg/metrics/collector/pod_ip_metrics_test.go

```diff
@@ -32,6 +33,68 @@ func mustCreateFile(t *testing.T, dir, name, content string) {
 	}
 }
 
+func mustCreateJSON(t *testing.T, dir, cidr string) {
+	path := dir + "/subnet"
+	data, err := os.ReadFile(path)
+	if err != nil {
+		data = []byte(`{
+			"type": "dummy",
+			"name": "gke-pod-network",
+			"cniVersion": "0.3.1",
+			"plugins": [
+				{
+					"type": "gke",
+					"mtu": 1460,
+					"ipam": {
+						"type": "host-local",
+						"ranges": [],
+						"routes": [
+							{"dst": "0.0.0.0/0"},
+							{"dst": "::/0"}
+						]
+					}
+				},
+				{
+					"type": "portmap",
+					"capabilities": {
+						"portMappings": true
+					}
+				},
+				{
+					"type": "cilium-cni"
+				}
+			]
+		}`)
+	}
+	var parsedData map[string]interface{}
+	if err := json.Unmarshal(data, &parsedData); err != nil {
```

Comment:
It may be complicating the test logic too much here with json manipulating - could we just put the ranges into the `data` var direclty?

Reply:
Done